src/dmtcp_launch.cpp

```diff
@@ -40,6 +40,7 @@ static int testJava(const char **argv);
 static bool testSetuid(const char *filename);
```

Comment:
Please modify the commit message.  I think you intended:
`Added --mpi flag to dmtcp_launch.`

Reply:
Fixed.